mne/viz/misc.py

```diff
@@ -311,19 +312,19 @@ def _plot_mri_contours(mri_fname, surf_fnames, orientation='coronal',
         ax.axis('off')
 
         # and then plot the contours on top
-        for surf in surfs:
+        for surf, color in surfs:
             if orientation == 'coronal':
                 ax.tricontour(surf['rr'][:, 0], surf['rr'][:, 1],
                               surf['tris'], surf['rr'][:, 2],
-                              levels=[sl], colors='yellow', linewidths=2.0)
+                              levels=[sl], colors=color, linewidths=2.0)
             elif orientation == 'axial':
                 ax.tricontour(surf['rr'][:, 2], surf['rr'][:, 0],
                               surf['tris'], surf['rr'][:, 1],
-                              levels=[sl], colors='yellow', linewidths=2.0)
+                              levels=[sl], colors=color, linewidths=2.0)
             elif orientation == 'sagittal':
                 ax.tricontour(surf['rr'][:, 2], surf['rr'][:, 1],
                               surf['tris'], surf['rr'][:, 0],
-                              levels=[sl], colors='yellow', linewidths=2.0)
+                              levels=[sl], colors=color, linewidths=2.0)
```

Comment:
Couldn't the coronal/axial/sagittal branching be reduced to make this a bit more concise? E.g.,

```
one = 0 if orientation == 'coronal' else 1
two = dict(coronal=2, axial=1, sagittal=2)[orientation]
ax.tricontour(surf['rr'][:, 2], surf['rr'][:, one],
     surf['tris'], surf['rr'][:, two],
     levels=[sl], colors=color, linewidths=2.0)
```